pkgs/applications/misc/gkrellm/default.nix

```diff
@@ -30,6 +43,11 @@ stdenv.mkDerivation rec {
   makeFlags = [ "STRIP=-s" ];
   installFlags = [ "DESTDIR=$(out)" ];
 
+  postInstall = ''
+    install -Dm444 -t $out/share/applications ${desktopItem}/share/applications/*
```

Comment:
@SuperSandro2000 showed me that we can also use `copyDesktopItems`, as for instance [here](https://github.com/NixOS/nixpkgs/blob/master/pkgs/games/freedroid/default.nix).

Reply:
Thanks, I wasn't aware of this.  Done.